### DISCUSSION TITLE
Notification center doesn't let you interact with notification

### DISCUSSION THREAD

**Discussion by raspin0** [linuxdeepin/developer-center#2901] [Features Request & Ideas | 特性请求 & 头脑风暴]
When you receive a notification sometimes you can click on it and something will happen. However if you miss the opportunity and notification is transferred into notification center it's dismissed on click. Please change this behavior so only clicking an "X" in the top-right corner of a notification is functional.

Also even native deepin apps suffer from this behaviour. When you take a screenshot and click "View" in notification center, it sometimes doesn't work and only dismisses notification.

**Comment by raspin0** [comment]
Um, sorry it seems it has been already patched out. I can't find a way to remove this thread, so please close it for me.

**Reply by BLumia** [reply]
It's fine to leave the thread here since it's not an issue, I'll mark it as resolved then, and thanks for the feedback :)
